applications/nrf_desktop/src/modules/hids.c

```diff
@@ -37,7 +37,10 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_HIDS_LOG_LEVEL);
  * Bluetooth LE connection event.
  */
 #define HIDS_SUBSCRIBER_PIPELINE_SIZE 0x02
-#define HIDS_SUBSCRIBER_REPORT_MAX    UINT8_MAX
+#define HIDS_SUBSCRIBER_REPORT_MAX    CONFIG_DESKTOP_HIDS_SUBSCRIBER_REPORT_MAX
+
```

Comment:
build assert regarding the relation with CONFIG_BT_ATT_TX_COUNT (it should be at least equal to the DESKTOP_HIDS_SUBSCRIBER_REPORT_MAX + maybe ATT response)

Reply:
Added with a comment